test/integration/Templates.test.ts

```diff
@@ -2,14 +2,16 @@ import * as postmark from '../../src/index';
 
 import { expect } from 'chai';
 import 'mocha';
-import { CreateTemplateRequest } from '../../src/client/models';
+import {CreateTemplateRequest, TemplatesPushRequest} from '../../src/client/models';
 
 const nconf = require('nconf');
 const testingKeys = nconf.env().file({ file: __dirname + '/../../testing_keys.json' });
 
 describe('Client - Templates', function () {
     const serverToken: string = testingKeys.get('SERVER_TOKEN');
+    const accountToken: string = testingKeys.get('ACCOUNT_TOKEN');
     let client = new postmark.ServerClient(serverToken);
+    let accountClient = new postmark.AccountClient(accountToken);
```

Comment:
`accountClient` seems to be immutable so `const` would be more appropriate

Reply:
fixed